pkg/bean/app.go

```diff
@@ -206,6 +206,23 @@ const (
 	WEBHOOK_EVENT_NON_MERGED_ACTION_TYPE string = "non-merged"
 )
 
+type CiPatchStatus string
+
+const (
```

Comment:
Can we use these constants from some common place

Reply:
Couldn't find the Not authorised.